src/io/flutter/view/FlutterView.java

```diff
@@ -237,6 +238,16 @@ protected void perform(AnActionEvent event) {
   }
 }
 
+class ToggleInspectModeAction extends AbstractToggleableAction {
```

Comment:
It is toggle-able. The device won't be completely out of inspect mode until the inspector is turned off either by flutter tool or intellij